.github/workflows/release.yml

```diff
@@ -0,0 +1,129 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  cancel_running_workflows:
+    name: Cancel running workflows
+    runs-on: ubuntu-22.04
+    steps:
+      - name: cancel running workflows
+        uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          access_token: ${{ github.token }}
+
+  build_deb:
```

Comment:
I see we have these build steps in https://github.com/radixdlt/babylon-node/blob/feature/DO-1166-deploy-dockerhub/.github/workflows/docker.yml#L21. So you can add another job there in that gets runs for release trigger.

Reply:
updated this